rules/incremental_load.sh.tpl

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env bash
+#
```

Comment:
Is this different from https://github.com/bazelbuild/rules_docker/blob/master/container/incremental_load.sh.tpl?

Reply:
Yes, significantly, the pointed template works with bazel Starlark rules that have a runfiles directory with resources available. This version works with repository rules that do not have a runfiles directory, but instead do have access to absolute paths.

Reply:
removed use of puller and this template